api/app/Http/Controllers/ApiController.php

```diff
@@ -43,6 +47,20 @@ public function getAll()
         return $this->getResponder()->collection($this->getRepository()->all());
     }
 
+    public function getAllPaginated(PaginatedRequestDecoratorInterface $request)
+    {
+        $count = $this->getRepository()->count();
+        $limit = $request->getLimit(static::PAGINATOR_DEFAULT_LIMIT, static::PAGINATOR_MAX_LIMIT);
+        $offset = $request->isGetLast()?$count-$limit:$request->getOffset();
+        //eloquent do not allow 0 or negative limit and offset
+        if ($limit <= 0) {
+            $collection = new Collection();
+        } else {
+            $collection = $this->getRepository()->all(['*'], $offset, $limit);
+        }
+        return $this->responder->paginatedCollection($collection, $offset, $count);
+    }
```

Comment:
The preference would certainly be to omit the content column for performance, however as it would be a limited query then there wouldn't be too much data coming back from the database.